src/posts/keymaps/peterjc-hands_down_promethium.md

```diff
@@ -0,0 +1,25 @@
+---
+author: peterjc
+baseLayouts: ["Hands Down"]
+firmwares: ["Karabiner-Elements"]
+hasHomeRowMods: false
+hasLetterOnThumb: true
+hasRotaryEncoder: false
+isAutoShiftEnabled: false
+isComboEnabled: true
+isSplit: false
+isTapDanceEnabled: false
+keybindings: []
+keyboard: Japanese Apple KeyBoard
+keyCount: 35
+keymapImage: https://codeberg.org/peterjc/kana-chording-ke/media/branch/main/hands-down-on-jis-macbook.jpeg
+keymapUrl: https://codeberg.org/peterjc/kana-chording-ke/src/branch/main/hands-down-on-jis-macbook.md
```

Comment:
This could be the raw JSON file?

https://codeberg.org/peterjc/kana-chording-ke/src/branch/main/hands-down-on-jis-macbook.json

Reply:
As per the DB fields reference document, the `keymapUrl` should be "[t]he link to the keymap folder files." Consequently, I am rather expecting a link to https://codeberg.org/peterjc/kana-chording-ke although I realize that the repo contains code for other related sub-projects. If you'd rather not confuse the reader by lots of irrelevant files, you can consider making a folder for hands-down-on-jis-macbook, move all relevant files there and then use a direct link to that newly created folder instead of the top-level repo root.

Reply:
Switched to the sub-folder structure as suggested.